algolia.gemspec

```
@@ -35,7 +35,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rake'
   spec.add_development_dependency 'rubocop', '<= 0.82.0'
 
-  spec.add_dependency 'faraday', ['>= 0.15', '< 2.0']
+  spec.add_dependency 'faraday', ['>= 2', '< 3']
+  spec.add_dependency 'faraday-net_http_persistent', '~> 2.0'
```

If we go that route, we should probably have the same range for this dependency
```suggestion
  spec.add_dependency 'faraday-net_http_persistent', ['>= 0.15', '< 3']
```